config/claude/pushover.sh

```
@@ -101,17 +100,17 @@ if echo "$input" | jq -e '.trigger' >/dev/null 2>&1; then
-# Handle SubagentStop hook
+# Handle SubagentStop hook (priority 0 = normal)
 if echo "$input" | jq -e '.stop_hook_active' >/dev/null 2>&1; then
-  send_notification "🤖 Subagent task completed" -1
+  send_notification "🤖 Subagent task completed" 0
   exit 0
 fi
 
-# Handle Stop hook (main session completion)
+# Handle Stop hook (priority 1 = high - Claude finished, needs attention)
 if echo "$input" | jq -e '.session_id' >/dev/null 2>&1; then
   SESSION_ID=$(echo "$input" | jq -r '.session_id[0:8]')
```

Shellcheck (0.11.0)

[warning] SESSION_ID appears unused. Verify use (or export if used externally).

(SC2034)